web/srv/server.go

```diff
@@ -173,6 +173,7 @@ func NewServer(
 	server.router.GET("/api/routes", handler.handleAPITopRoutes)
 	server.router.GET("/api/edges", handler.handleAPIEdges)
 	server.router.GET("/api/check", handler.handleAPICheck)
+	server.router.GET("/api/namespaces/:namespace/:resource-type/:resource-name", handler.handleAPIResourceDefinition)
```

Comment:
This API where we simply can look up resources by name seems good but it's hard to know whether it will be sufficient without knowing the use-cases.

For example, when visiting a service detail page, we might also want to look up the service profile for that service.  And possibly all the service profiles for that service from other namespaces.  Or we may want to look up all traffic splits which have this service as an apex or leaf.  Such lookups would require a more specialized API.

Reply:
API has been updated to `/api/resource-definition` and uses now query string params instead of path params 👍